### ISSUE TITLE
remove javascript.use_us_english_locale ???

### ISSUE THREAD

**Issue by atomGit** [ghacksuserjs/ghacks-user.js#764]
`javascript.use_us_english_locale` - apparently RFP sets the language, but i don't know if it affects this pref specifically - if is does, can this be removed?

**Comment by Thorin-Oakenpants** [comment]
can't remove them: people may not use RFP

If you are already on en-US then when you turn on RFP, nothing happens. I need to check in the second case:
- A: `en-US` download/install
- B: `non en-US` and you already flipped things to `en-US`

If you're not en-US, then RFP asks you if you want to spoof your language to websites and sets a preference `privacy.spoof_english`: depending on your answer, it will change some of the other prefs
- A: `fr` download/install
- B: `non fr` and you already flipped things to `fr`

Again, I need to check what happens in case b

To maybe make matters even more tricky, language options are changing in the UI, and how they determine the language send in headers (request websites in a language) is also changed. I think it was about 69+, with more to come. Instead of an option, it now takes the topmost language in your list, and the option in the UI for the language to use in the application (menus, messages from Firefox, i.e non-web) is gone.

And, coming soon, you will be able to add app language packs on the fly: and I do not know how this will affect the language fingerprint: since you know, `navigator.languages` (plural) is a thing

It's all very messy, and I haven't mapped it out completely, which is why I have #730 set up - just been waiting for things to land and for when I have time. I don't want to make it a hassle for users to try and use a language of their choice, but I also need to understand it.

Options:
currently all active
- so a non en-US user gets RFP flipped on by this user.js: because privacy.spoof_english does not exist, it gets created with I think a value of zero (I think: ignore the values: there's three of them: a tri-state) and a prompt is shown, user chooses ask for english or ask for current app language (and the soof english pref is set as 1 or 2, and based on the value, other prefs may get set)
- lets say the non en-US user says, nah, let me request pages in my native Klingon, then RFP spoof privacy is set as non-zero (so you don't get asked again), and then the user.js does it for them anyway on the next FF start: so we're not respecting the end-user's choices

all inactive
- because we set RFP and non en-US users **should** get a prompt. en-US users are already covered (but what about additional languages they install? fuuuck)

---

I had another complete matrix of results to lay out here, but this is doing my head in a bit right now... just woke up and I need a coffee ...

long story short: I think, but have not decided, that the three (or four?) items in question can be inactive and flipped on for non-RFP and non-en-US users if they want. en-US users are already fine. That simplifies it to two groups: prefs inactive
- group A: en-US users (already covered), plus non-en-US users who answered the RFP prompt (their choice is respected)
- group B: non en-US users without RFP (not covered)

given group B is more likely to want their own native language, why force english on them. Then again, why not enforce everything, so users have to consciously MAKE A CHOICE and understand why: overrides

So now I'm leaning on enforcing it. It also covers some holes by resetting them on start, I think. especially with upcoming changes

.... COFFEEEEEEE
